src/SME.SGP.Dados/Repositorios/RepositorioFechamentoTurmaConsulta.cs

```diff
@@ -58,6 +58,7 @@ public async Task<FechamentoTurma> ObterPorTurmaCodigoBimestreAsync(string turma
             var query = new StringBuilder(@"select f.* 
                             from fechamento_turma f
                           inner join turma t on t.id = f.turma_id
+                                INNER JOIN conselho_classe cc ON cc.fechamento_turma_id  = f.id 
```

Comment:
Realmente é inner? Não seria melhor um left?

Reply:
se for left ele pode retornar um fechamento turma sem conselho_classe  e na hora de salvar ele precisar do 
conselhoclasseId
por isso estava dando erro na tela na hora de salvar por é feito um QueryFirstOrDefaultAsync em uma lista 
e nem sempre o primeiro da lista tem conselhoclasseId com left

Reply:
Thiago... acho que vc esta considerando só o cenário atual.
Esse método é referenciado mais de 10x e tem casos que vc precisa retornar os dados do fechamento, Independente de existir conselho de classe, afinal é o que retorna esse método Obter(Fechamento)PorTurmaCodigoBimestre